recipes/uutils/meta.yaml

```
@@ -0,0 +1,56 @@
+{% set name = "uutils" %}
+{% set version = "0.0.16" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/uutils/coreutils/archive/{{ version }}.tar.gz
+  sha256: 1bdc2838e34b6ca8275cb8e3bac5074ca59158ea077e3195fb91ec0cc6b594f6
+  patches:
+    - 0001-FIX_BUILDDIR_IN_MAKE.patch  # [not win]
+    # Fixes long command and quotes problems
+    - 0002-SHORTEN_SH_COMMANDS.patch
+
+build:
+  number: 0
+  ignore_run_exports:
+    - libcxx-ng
```

```suggestion
  ignore_run_exports_from:
    - {{ compiler('cxx') }}
```
The cxx standard library is also not used.